connectors/http/src/main/java/quarks/connectors/http/HttpStreams.java

```diff
@@ -38,12 +40,88 @@ Licensed to the Apache Software Foundation (ASF) under one
  */
 public class HttpStreams {
     
+    /**
+     * Make an HTTP GET request with JsonObject. <br>
+     * 
+     * Method specifically works with JsonObjects. For each JsonObject in the stream, 
+     * HTTP GET request is executed on provided uri. As a result, Response is added to
+     * the response TStream.
+     * <br>
+     * 
+     * Sample usage:<br>
+     * 
+     * <pre>
+     * {@code
+     *     JsonObject request1 = new JsonObject();
+     *     request1.addProperty("a", "abc");
+     *     request1.addProperty("b", "42");
+     *     TStream<JsonObject> rc = HttpStreams.getJson(
+     *             topology.collection(Arrays.asList(request1)),
+     *             HttpClients::noAuthentication,
+     *             t -> url + "a=" + t.get("a").getAsString() + "&b="
```

Comment:
I think it may be clearer for the user if a declaration of the `url` variable is added.